src/vl-rich-data.js

```diff
@@ -0,0 +1,380 @@
+ * @property {boolean} data-vl-filter-closed - Attribuut dat aangeeft of dat de filter gesloten is.
+ *
+ * @slot toggle-filter-button-text - slot om de tekst te kunnen wijzigen van de toggle filter knop. Default: Filter.
+ * @slot close-filter-button-text - slot om de onzichtbare tekst te kunnen wijzigen van de filter sluit knop. Default: Filter sluiten.
+ *
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-rich-data/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-rich-data/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-rich-data.html|Demo}
+ */
+export class VlRichData extends vlElement(HTMLElement) {
+  static get _observedAttributes() {
+    return ['data', 'collapsed-m', 'collapsed-s', 'collapsed-xs', 'filter-closable', 'filter-closed'];
+  }
+
+  static get _defaultSearchColumnSize() {
+    return 4;
+  }
+
+  constructor(style = '', content = '') {
+    super(`
+      <style>
+        @import "/src/style.css";
+        @import "/node_modules/vl-ui-grid/dist/style.css";
+        @import "/node_modules/vl-ui-icon/dist/style.css";
+        @import "/node_modules/vl-ui-button/dist/style.css";
+      </style>
+      ${style}
+      <div>
+        <div is="vl-grid" is-stacked>
+          <div id="toggle-filter" is="vl-column" class="vl-u-align-right vl-u-hidden--s" hidden size="12">
+            <button id="toggle-filter-button" is="vl-button-link" type="button" aria-label="Toon de filter">
+              <span is="vl-icon" data-vl-icon="content-filter" data-vl-before></span><slot name="toggle-filter-button-text">Filter</slot>
+            </button>
+          </div>
+          <div id="open-filter" is="vl-column" class="vl-u-align-right vl-u-hidden" hidden size="12">
+            <button id="open-filter-button" is="vl-button-link" type="button" aria-label="Toon de filter">
+              <span is="vl-icon" data-vl-icon="content-filter" data-vl-before></span><slot name="toggle-filter-button-text">Filter</slot>
+            </button>
+          </div>
+          <div id="search" is="vl-column" size="0" small-size="0">
+            <button id="close-filter-button" class="vl-filter__close" hidden type="button">
+              <span is="vl-icon" data-vl-icon="close"></span>
+              <span class="vl-u-visually-hidden"><slot name="close-filter-button-text">Filter sluiten</slot></span>
+            </button>
+            <div id="filter-slot-container">
+              <slot id="filter-slot" name="filter"></slot>
+            </div>
+          </div>
+          <div id="content" is="vl-column" size="12">
+            <div is="vl-grid" is-stacked>
+              <div id="search-results" is="vl-column" size="8" small-size="6">
+                <span>We vonden</span>
+                <strong><span id="search-results-number">0</span> resultaten</strong>
+              </div>
```

Comment:
Indien er geen resultaten gevonden worden, dit ook aangeven met bv. "We vonden geen resultaten". Momenteel wordt er NaN getoond wanneer de api faalt en de data niet gezet is. 
Zie screenshot ter voorbeeld. 
<img width="1120" alt="Screenshot 2020-07-08 at 13 03 25" src="https://user-images.githubusercontent.com/8820456/86911334-74700e80-c11b-11ea-8780-947db61b31ff.png">

Reply:
ik zag dat hier ook al een alternatief voor gelogged was https://github.com/milieuinfo/webcomponent-vl-ui-rich-data-table/issues/60

Reply:
Het alternatief is om nog een extra melding toe te voegen om te tonen aan de gebruiker, ik heb de fallback naar 0 alvast in kader van dit issue voorzien.